pyensembl/database.py

```diff
@@ -81,25 +82,42 @@ def _create_database(self):
             indices=indices)
         return db
 
-    def _connect_or_create_database(self):
+    def connection_if_exists(self):
```

Comment:
It seems a little odd that `_connect_or_create_database` is still "private" but this method is "public" (doesn't start with an underscore). Seems like they should be equally exposed to the user (in either direction).